src/BlazorUI/Bit.BlazorUI/Components/DatePicker/BitDatePicker.razor

```diff
@@ -60,47 +60,47 @@
                  role="group"
                  aria-label="">
                 <div aria-live="polite" aria-atomic="true">Selected date @CurrentValueAsString</div>
-                @if ((showMonthPickerAsOverlayInternal && !isMonthPickerOverlayOnTop) || !showMonthPickerAsOverlayInternal)
+                @if ((_showMonthPickerAsOverlayInternal && !_isMonthPickerOverlayOnTop) || !_showMonthPickerAsOverlayInternal)
                 {
                     <div class="day-picker-wrapper">
                         <div class="picker-header">
-                            @if (showMonthPickerAsOverlayInternal)
+                            @if (_showMonthPickerAsOverlayInternal)
                             {
-                                <button class="header-title" aria-label="@($"{monthTitle}, change month")"
-                                        aria-live="polite" aria-atomic="true" tabindex="0" type="button" @onclick="@ToggleMonthPickerAsOverlay">
-                                    <span id="@MonthAndYearId">@monthTitle</span>
+                                <button class="header-title" aria-label="@($"{_monthTitle}, change month")"
+                                aria-live="polite" aria-atomic="true" tabindex="0" type="button" @onclick="@ToggleMonthPickerAsOverlay">
+                                    <span id="@MonthAndYearId">@_monthTitle</span>
                                 </button>
                             }
                             else
                             {
                                 <div class="header-title" aria-live="polite" aria-atomic="true" tabindex="-1">
-                                    <span id="@MonthAndYearId">@monthTitle</span>
+                                    <span id="@MonthAndYearId">@_monthTitle</span>
                                 </div>
                             }
                             <div class="navigation-btn-container">
                                 <button class="navigation-btn"
-                                        type="button"
-                                        title="@($"Go to previous month")"
-                                        aria-disabled="@CheckMonthForMaxAndMinDate(ChangeDirection.Previous)"
-                                        disabled="@CheckMonthForMaxAndMinDate(ChangeDirection.Previous)"
-                                        @onclick="() => HandleMonthChange(ChangeDirection.Previous)">
+                                    type="button"
+                                    title="@($"Go to previous month")"
+                                    aria-disabled="@IsMonthOutOfMinAndMaxDate(ChangeDirection.Previous)"
+                                    disabled="@IsMonthOutOfMinAndMaxDate(ChangeDirection.Previous)"
+                                    @onclick="() => HandleMonthChange(ChangeDirection.Previous)">
```

Comment:
please correct all of these misformatted codes everywhere. unfortunately, VS has so many bugs in this area and you need to do it manually for now.